### PR TITLE
fix: submission too long has the wrong message

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** [ Fix the error message for when an ORA submission is too long ]

JIRA: [AU-559](https://openedx.atlassian.net/browse/AU-559)

**What changed?**

- Change the error message when an ORA is too long to be about that instead of multiple submissions
- Add a note about autocorrect

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

- In an ORA, submit something super long. Like ... the entire text of moby dick https://www.gutenberg.org/files/2701/2701-h/2701-h.htm and see the new error message

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/xblock/submission_mixin.py [hunk 1]
```diff
@@ -171,6 +171,13 @@ def submit(self, data, suffix=''):  # pylint: disable=unused-argument
                 )
                 if answer_too_long:
                     status_tag = 'EANSWERLENGTH'
+                    max_size = f"({int(api.Submission.MAXSIZE / 1024)} KB)"
+                    base_error = self._("Response exceeds maximum allowed size.")
+                    extra_info = self._(
+                        "Note: if you have a spellcheck or grammar check browser extension, "
+                        "try disabling, reloading, and reentering your response before submitting."
+                    )
+                    status_text = f"{base_error} {max_size} {extra_info}"
```
Comment (nsprenkle) [inline openassessment/xblock/submission_mixin.py:180]: This code path doesn't log the error. Is that something we want to have logged for our tracking?

**Review by nsprenkle — COMMENTED** [review]
Note about logging. Also, is there a reason that package-lock and one of the djangojs files are mostly empty?

File: openassessment/xblock/submission_mixin.py (hunk 1 above)
Comment (jansenk) [reply openassessment/xblock/submission_mixin.py:180]: Yep, missed that

**Comment by jansenk** [comment]
> is there a reason that package-lock and one of the djangojs files are mostly empty?

Good catch? I have no clue. I'll try redoing translations and see if it fixes it

File: openassessment/xblock/submission_mixin.py [hunk 2]
```diff
@@ -170,7 +170,17 @@ def submit(self, data, suffix=''):  # pylint: disable=unused-argument
                     for answer_err in err.field_errors.get('answer', [])
                 )
                 if answer_too_long:
+                    logger.exception(
+                        f"Attempted to submit response for {student_item_dict} but the submission is too long"
+                    )
```
Comment (nsprenkle) [inline openassessment/xblock/submission_mixin.py:175]: Nit: putting the student item dict at the end of the log message would improve search-ability and proximity to the surfaced message would help as well.

```suggestion
                    logger.exception(
                        f"Response exceeds maximum allowed size: {student_item_dict}"
                    )
```

**Review by nsprenkle — APPROVED** [review]
Looks good, with a suggestion for improved log messaging

File: openassessment/xblock/submission_mixin.py (hunk 2 above)
Comment (jansenk) [reply openassessment/xblock/submission_mixin.py:175]: I was about to get sarcastically annoyed because at firt I thought I'd have to re-do the translations nonsense but then I realized this is a log message and not translated! wahoo
